sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/tests/OpenTelemetry.Exporter.AzureMonitor.IntegrationTests.WebApp/OpenTelemetry.Exporter.AzureMonitor.Integration.Tests.WebApp.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <!-- 
+  Integration tests evaluate an app's components on a broader level than unit tests. 
+  Unit tests are used to test isolated software components, such as individual class methods. 
+  Integration tests confirm that two or more app components work together to produce an expected result, 
+  possibly including every component required to fully process a request.
+  https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-3.1
```

Comment:
nit: sanitize the URL (e.g. removing `en-us` and the `?view=aspnetcore-3.1` parts).